smart-table-module/js/Table.js

```diff
@@ -25,6 +25,9 @@
             scope.currentPage = 1;
             scope.holder = {isAllSelected: false};
 
+            // index of last row that was selected
+            this.prevouslySelectedIndex = null;
```

Comment:
Ahh, good catch! Haha. 

I use it in turntable-contextual-menu (which I will create a separate PR for). I technically don't have to declare it here, but I did it in order to indicate that this is available and being used (similar to how sortAlgorithm and filterAlgorithm variables are indicated above as well).